taglib/fileref.cpp

```diff
@@ -54,6 +54,62 @@
 
 using namespace TagLib;
 
+template <typename T>
+static File* createInternal(T arg, const String& ext, bool readAudioProperties,
```

Comment:
Just wanted to check we had the same understanding :)
I just pushed that change as well, and updated the tests so all tests are also using the IOStream constructor.